src/modules/tracker.js

```diff
@@ -1228,6 +1228,398 @@ class Tracker {
+          return new Error('"item_id" must be a string');
+        }
+        bodyParams.item_id = item_id;
+      }
+
+      if (!helpers.isNil(item_name)) {
+        if (typeof item_name !== 'string') {
+          return new Error('"item_name" must be a string');
+        }
+        bodyParams.item_name = item_name;
+      }
+
+      if (!helpers.isNil(section)) {
+        if (typeof section !== 'string') {
+          return new Error('"section" must be a string');
+        }
+        queryParams.section = section;
+      }
+
+      if (!helpers.isNil(result_count)) {
+        if (typeof result_count !== 'number') {
+          return new Error('"result_count" must be a number');
+        }
+        bodyParams.result_count = result_count;
+      }
+
+      if (!helpers.isNil(result_id)) {
+        if (typeof result_id !== 'string') {
+          return new Error('"result_id" must be a string');
+        }
+        bodyParams.result_id = result_id;
+      }
+
+      if (!helpers.isNil(result_page)) {
+        if (typeof result_page !== 'number') {
+          return new Error('"result_page" must be a number');
+        }
+        bodyParams.result_page = result_page;
+      }
+
+      if (!helpers.isNil(num_results_per_page)) {
+        if (typeof num_results_per_page !== 'number') {
+          return new Error('"num_results_per_page" must be a number');
+        }
+        bodyParams.num_results_per_page = num_results_per_page;
+      }
+
+      if (!helpers.isNil(result_position_on_page)) {
+        if (typeof result_position_on_page !== 'number') {
+          return new Error('"result_position_on_page" must be a number');
+        }
+        bodyParams.result_position_on_page = result_position_on_page;
+      }
+
+      bodyParams.action_class = 'result_click';
+
+      const requestURL = `${requestPath}${applyParamsAsString(queryParams, this.options)}`;
+      const requestMethod = 'POST';
+      const requestBody = applyParams(bodyParams, { ...this.options, requestMethod });
+
+      this.requests.queue(
+        requestURL,
+        requestMethod,
+        requestBody,
+        networkParameters,
+      );
+      this.requests.send();
+
+      return true;
+    }
+
+    this.requests.send();
+
+    return new Error('parameters are required of type object');
+  }
+
+  /**
+   * Send quiz conversion event to API
+   *
+   * @function trackQuizConversion
+   * @param {object} parameters - Additional parameters to be sent with request
+   * @param {string} parameters.quiz_id - Quiz Id
+   * @param {string} parameters.quiz_version_id - Quiz version Id
+   * @param {string} [parameters.item_id] - Product item unique identifier (Either item_id or item_name is required)
+   * @param {string} [parameters.item_name] - Product item name
+   * @param {string} [parameters.variation_id] - Product item variation unique identifier
+   * @param {string} [parameters.revenue] - The subtotal (not including taxes, shipping, etc.) of the entire order
```

Comment:
I think this line uses the description of trackPurchase but it should be using the description of trackConversion since this page would have individual items not an order with multiple items

Reply:
Good catch